src/SwedbankPay.Sdk/Payments/CardPayments/CardPaymentOperations.cs

```diff
@@ -0,0 +1,80 @@
+                        RedirectAuthorization = httpOperation;
+                        break;
+
+                    case PaymentResourceOperations.ViewAuthorization:
+                        ViewAuthorization = httpOperation;
+                        break;
+
+                    case PaymentResourceOperations.DirectAuthorization:
+                        DirectAuthorization = async payload =>
+                            await client.SendAsJsonAsync<CardPaymentAuthorizationResponse>(httpOperation.Method, httpOperation.Href, payload);
+                        break;
+
+                    case PaymentResourceOperations.CreateCapture:
+                        Capture = async payload =>
+                            await client.SendAsJsonAsync<CaptureResponse>(httpOperation.Method, httpOperation.Href, payload);
+                        break;
+
+                    case PaymentResourceOperations.CreateCancellation:
+                        Cancel = async payload =>
+                            await client.SendAsJsonAsync<CancellationResponse>(httpOperation.Method, httpOperation.Href, payload);
+                        break;
+
+                    case PaymentResourceOperations.CreateReversal:
+                        Reverse = async payload =>
+                            await client.SendAsJsonAsync<ReversalResponse>(httpOperation.Method, httpOperation.Href, payload);
+                        break;
+
+                    case PaymentResourceOperations.RedirectVerification:
+                        RedirectVerification = httpOperation;
+                        break;
+
+                    case PaymentResourceOperations.ViewVerification:
+                        ViewVerification = httpOperation;
+                        break;
+
+                    case PaymentResourceOperations.DirectVerification:
+                        DirectVerification = httpOperation;
+                        break;
+
+                    case PaymentResourceOperations.PaidPayment:
+                        PaidPayment = httpOperation;
+                        break;
+                }
+                this.Add(httpOperation.Rel, httpOperation);
+            }
+        }
+
+        public Func<CardPaymentCancelRequest, Task<CancellationResponse>> Cancel { get; }
+        public Func<CardPaymentCaptureRequest, Task<CaptureResponse>> Capture { get; }
+        public Func<CardPaymentAuthorizationRequest, Task<CardPaymentAuthorizationResponse>> DirectAuthorization { get; }
+        public Func<CardPaymentReversalRequest, Task<ReversalResponse>> Reverse { get; }
+        public HttpOperation DirectVerification { get; internal set; }
+        public HttpOperation PaidPayment { get; internal set; }
+        public HttpOperation RedirectAuthorization { get; internal set; }
+        public HttpOperation RedirectVerification { get; internal set; }
+        public HttpOperation Update { get; internal set; }
```

Comment:
Why isn't the `Update` operation a `Func`?

Reply:
No idea, I can look into it.